### ISSUE TITLE
OSGi: MongoJack v5.0.0 cannot be resolved due to wrong value of osgi.ee filter value in Require-Capability header 

### ISSUE THREAD

**Issue by rakeshk15** [mongojack/mongojack#246]
MongoJack v5.0.0 cannot be resolved because it is expecting the osgi.ee capability with "UNKNOWN" value which is wrong.

Here is the Require-Capability header from manifest

`Require-Capability: osgi.ee; filter:="(osgi.ee=UNKNOWN)"`

A valid example of osgi.ee filter 

`Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"`

**Comment by rakeshk15** [comment]
Maven bundle plugin should be updated to the latest version to solve this issue.

I tried to build the project using v5.1.9 of maven-bundle-plugin and the osgi.ee filter was correctly generated.

**Comment by rlodge** [comment]
Published a 5.0.1-SNAPSHOT to oss.sonatype.org.  If you can verify that it works, I'll cut a release.

**Comment by rakeshk15** [comment]
Thanks @rlodge, i'll pull in the snapshot and test.

Also, there is one more thing which needs to be looked into.

MongoJack expects SLF4J v2.0.12 packages but as of now most prevalent version is 1.7.36 and due to the unavailability of SLF4J v2 in OSGi runtime the bundle remains unresolved.

If we import a range then this issue can be solved.

For this an Import-Package header should be added in maven-bundle-plugin configuration.

`Import-Package: org.slf4j;version="[1.7,3)"`

**Comment by rakeshk15** [comment]
Here is the working plugin declaration.

    ```
<plugin>
        <groupId>org.apache.felix</groupId>
        <artifactId>maven-bundle-plugin</artifactId>
        <version>5.1.9</version>
        <extensions>true</extensions>
        <configuration>
          <instructions>
            <Export-Package>org.mongojack</Export-Package>
            <Import-Package>org.slf4j;version="[1.7,3)",*</Import-Package>
          </instructions>
        </configuration>
        <executions>
          <execution>
            <id>bundle-manifest</id>
            <phase>process-classes</phase>
            <goals>
              <goal>manifest</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
```

**Comment by rlodge** [comment]
Pushed another snapshot build.

**Comment by rakeshk15** [comment]
Hi @rlodge 5.0.1-SNAPSHOT looks good.

Please go ahead with release when you get time.

**Comment by rlodge** [comment]
Released 5.0.1.  Might take a bit for it to propagate to central.  Let me know if it doesn't show up.
